hugr-py/src/hugr/node_port.py

```diff
@@ -28,6 +28,8 @@ class Direction(Enum):
 
 NodeIdx = int
 PortOffset = int
+Edge = tuple["OutPort", "InPort"]
```

Comment:
Is this related? (Neither of these is re-exported)

Reply:
Not at all :P
It was part of the port of useful things from guppy, but I should probably remove it from here.